### ISSUE TITLE
Example wifiManager

### ISSUE THREAD

**Issue by m0rt3nlund** [JiriBilek/WiFiSpi#11]
Hi!

I was just wondering if you could make an example for how you would implement the clientcode to support the ESP running wifiManager?

This appears to stop your code running on the ESP until a wifi connection is established? And therefor it does not respond to the client during this time?

Mortrn

**Comment by JiriBilek** [comment]
Yes, it stops the client. Obviously, it must be stopped to be updated OTA. I haven't tried the code yet, it's a shame :(. Maybe jandrassy could help you, it's his code.
J.
